plugins/scaffolder-backend/src/scaffolder/stages/publish/publishers.ts

```diff
@@ -80,21 +80,19 @@ export class Publishers implements PublisherBuilder {
     const githubConfig = config.getOptionalConfig('scaffolder.github');
     if (githubConfig) {
       try {
-        const repoVisibility = githubConfig.getString(
-          'visibility',
-        ) as RepoVisibilityOptions;
-
+        const repoVisibility = githubConfig.getString('visibility');
         const githubToken = githubConfig.getString('token');
         const githubHost =
           githubConfig.getOptionalString('host') ?? 'https://api.github.com';
+
         const githubClient = new Octokit({
           auth: githubToken,
           baseUrl: githubHost,
         });
         const githubPublisher = new GithubPublisher({
           client: githubClient,
           token: githubToken,
-          repoVisibility,
+          repoVisibility: repoVisibility as any,
```

Comment:
What's the reason to not use `as RepoVisibilityOptions` any longer?

Reply:
It was only used in one place which was in this repo, and I got this error

```
packages/backend/src/plugins/scaffolder.ts:13:3 - error TS2459: Module '"../../../../node_modules/@backstage/plugin-scaffolder-backend/dist"' declares 'RepoVisibilityOptions' locally, but it is not exported.
```

https://github.com/backstage/backstage/pull/4001/checks?check_run_id=1679687451

So I'm kinda just trying to get rid of that error frankly :) But it still happens after that change.

Reply:
It's used in a few places actually still, in some documentation too. Especially `create-app` thinking that we should put  it back.

https://github.com/backstage/backstage/blob/1ddcf191dbf5f0280383e3bb20f251016ed14889/packages/create-app/templates/default-app/packages/backend/src/plugins/scaffolder.ts#L13

https://github.com/backstage/backstage/blob/af141c0766db5536045a2f849d43214f151f9430/docs/features/software-templates/installation.md#L97

Reply:
Yeah I noticed, I'll try to make it work with them intact instead.